netlify.toml

```diff
@@ -1,363 +1,363 @@
 [[redirects]]
-  from = "/posts/8-this-is-our-story/"
```

Comment:
to be perfectly honest, I'd much rather keep the original urls with IDs for old posts, and keep the so-called inconsistency (I hesitate to call it that since it makes this sounds much worse than it is)

url's are ultimately not that important. the previous ones are readable as well, they just happen to have a now-meaningless number. It's not like anyone's reading the blog is gonna care

So, I can approve this, but I'd like to have someone else's input on this

Reply:
Fair enough, requested a voting in Slack (https://subvisual.slack.com/archives/C02A7GR2N/p1587462683009100).

Reply:
Voting closed, removing the redirects and keeping the `path` field.